go.mod

```diff
@@ -165,3 +165,7 @@ replace (
 
 // Exclude this version because the version has an invalid checksum.
 exclude github.com/docker/distribution v2.8.0+incompatible
+
+replace (
+	github.com/elastic/elastic-agent-libs => github.com/ycombinator/elastic-agent-libs v0.0.0-20230504062120-4ef5f2463ba0
+)
```

Comment:
These lines should be removed and the `require` directive for `elastic-agent-libs` should be update once https://github.com/elastic/elastic-agent-libs/pull/118 is merged and built.

Reply:
Removed in 219eadd81bb335211f015cbf7858efb946b941a3.